### PR TITLE
Show english transliteration in search result

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
image

### CODE REVIEW

**Comment by theamanjs** [comment]
@Gauravjeetsingh I believe if we capitalize the verse it would look better than now.

File: www/main/common/sttm-ui/search-results/SearchResults.jsx [hunk 1]
```diff
@@ -107,6 +109,7 @@ const SearchResults = ({
             )} ${ang} `}</span>
           )}
           <span className="gurmukhi">{highlightKeywords(verse, searchQuery)}</span>
+          {currentLanguage === 'en' && <div className="eng-verse">{anvaad.translit(verse)}</div>}
```
Comment (theamanjs) [inline www/main/common/sttm-ui/search-results/SearchResults.jsx:112]: veerji, recently I learned this. We should keep our static strings in a standard file. Just like 'en' all the strings from all the files should be moved to a javascript file. You can create this issue and work on this later.

File: www/src/scss/navigator/common/_verse-block.scss [hunk 2]
```diff
@@ -88,12 +88,17 @@
...
-
     div {
       display: inline-flex;
       a {
         padding: 5px;
         font-size: 1rem;
+
+        .eng-verse {
+          color: $rust;
+          display:block;
```
Comment (theamanjs) [inline www/src/scss/navigator/common/_verse-block.scss:99]: indentation issue

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: www/src/scss/navigator/common/_verse-block.scss (hunk 2 above)
Comment (Gauravjeetsingh) [reply www/src/scss/navigator/common/_verse-block.scss:99]: Looks fine to me veerji, it has two spaces everywhere.

**Pull request merged**